packages/admin/components/Layout/Layout.tsx

```diff
@@ -114,6 +114,7 @@ export const Layout: React.FC = ({ children }) => {
         <AntLayout
           css={css`
             margin-left: ${isSidebarCollapsed ? 80 : 200}px;
+            padding: 24px;
```

Comment:
Nešlo by zde použit em nebo rem?

Reply:
On je celý Ant napsaný v pixelech, takže aby to lícovalo, tak je dobré je také používat. 
U administrace není ta responzivita tak důležitá jako u webu.